Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM debian:wheezy
+MAINTAINER Di Wu <diwu@yelp.com>
+
+COPY . /opt/redash/current/
+
+# Install dependencies
+WORKDIR /opt/redash/current
+RUN bash setup/bootstrap_docker.sh
+
+# Build frontend sources
+WORKDIR /opt/redash/current/rd_ui
+USER redash
+RUN mkdir -p /opt/redash/.npm/node_modules
+RUN npm config set prefix /opt/redash/.npm/node_modules
+RUN npm install -g bower
+RUN npm install -g grunt-cli
+RUN npm install
+RUN /opt/redash/.npm/node_modules/bin/bower install
+RUN /opt/redash/.npm/node_modules/bin/grunt build
+
+# Reset working directory and user for future `docker attach` sessions
+WORKDIR /opt/redash/current
+USER root
+
+CMD service redash_supervisord start
```

Comment:
Ideally, it will be better replace supervisord with running multiple docker images, but we can keep it for now. But let's not run it daemonized -- you can run supervisord in foreground mode (with the `-n` argument).